Blogs/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+from .views import BlogList, BlogDetailView, BlogCreateView, MyBlogList
+
+# from .views import
+
+urlpatterns = [
+    path("create-blog/", BlogCreateView.as_view(), name="create-blog"),
```

Comment:
Change the URL names as per described in the document.

Reply:
> Change the URL names as per described in the document.

Sir, this is an additional feature as it is not asked in Django assignment document, but in context of this application, I felt it is quite necessary feature so I have added it to this application.